test/performance/select_perf_test.lua

```diff
@@ -0,0 +1,176 @@
+local fio = require('fio')
+local fiber = require('fiber')
+local net_box = require('net.box')
+local log = require('log')
+
+local t = require('luatest')
+local g = t.group('perf')
+
+local helpers = require('test.helper')
+
+math.randomseed(os.time())
```

Comment:
I don't understand why we need seed initialization here if we don't use random generator in a test.

Reply:
It's an ancient artifact. Removed.